cds_ils/migrator/relations/api.py

```diff
@@ -88,11 +90,15 @@ def migrate_siblings_relation():
                 relation["relation_type"]
             )
 
+            if relation_type == OTHER_RELATION.name:
+                extra_metadata.update(
```

Comment:
Are you using the `extra_metadata` somewhere?

Reply:
👍 nice ! good catch